Zend/zend_atomic.h

```diff
@@ -63,10 +58,9 @@ static zend_always_inline bool zend_atomic_bool_exchange_ex(zend_atomic_bool *ob
 	return InterlockedExchange8(&obj->value, desired);
 }
 
-/* On this platform it is non-const due to Iterlocked API*/
-static zend_always_inline bool zend_atomic_bool_load_ex(zend_atomic_bool *obj) {
+static zend_always_inline bool zend_atomic_bool_load_ex(const zend_atomic_bool *obj) {
 	/* Or'ing with false won't change the value. */
-	return InterlockedOr8(&obj->value, false);
+	return InterlockedOr8(&((zend_atomic_bool *) obj)->value, false);
 }
 
```

Comment:
I'm still against lying about const correctness here. On this platform it's _not_ const, sorry. I know it's not as clean as nice to have to detect which platforms it is and isn't const on, but that's just reality.

Reply:
I also have said something about that in #8881 :

> [bson-atomic](https://github.com/mongodb/mongo-c-driver/blob/master/src/libbson/src/bson/bson-atomic.h#L205) also used type casting to avoid such problems, to be on the safe side, I've added type-casts to all calls.

I think bson has also been proven in production extensively...